source/hotkeys_.d

```diff
@@ -135,6 +137,8 @@ string defaultName(Bindable bindable){
 		case turnRight: return "Turn Right";
 		case cameraZoomIn: return "Camera zoom in";
 		case cameraZoomOut: return "Camera zoom out";
+		case moveUp: return "Move Up";
+		case moveDown: return "Move Down";
```

Comment:
Should be moved to `SacEngine extensions` section.

Reply:
moved.